spec/unit/hanami/helpers/form_helper_spec.rb

```diff
@@ -2427,6 +2427,34 @@
         expect(actual).to include(%(<select name="book[store]" id="book-store">\n<option value="it" selected="selected">Italy</option>\n<option value="us">United States</option>\n</select>))
       end
     end
+
+    describe 'with nil as a value' do
+      let(:option_values) { Hash['Italy' => 'it', 'United States' => 'us', 'N/A' => nil] }
+
+      it "doesn't set options as selected" do
+        actual = view.form_for(:book, action) do
+          select :store, option_values
+        end.to_s
+
+        expect(actual).to include(%(<select name="book[store]" id="book-store">\n<option value="it">Italy</option>\n<option value="us">United States</option>\n<option value="" selected="selected">N&#x2F;A</option>\n</select>))
+      end
+
+      it "set as selected the option with nil value" do
```

Comment:
This has the same outcome as the previous case. But I guess it's fine to test them both...